CPAC/pipeline/engine.py

```diff
@@ -2503,14 +2552,33 @@ def ingress_pipeconfig_paths(cfg, rpool, unique_id, creds_path=None):
                 creds_path=creds_path,
                 dl_dir=cfg.pipeline_setup["working_directory"]["path"],
             )
-            rpool.set_data(
-                key,
-                config_ingress,
-                "outputspec.data",
-                json_info,
-                "",
-                f"{key}_config_ingress",
-            )
+            node = config_ingress
+            output = "outputspec.data"
+            node_name = f"{key}_config_ingress"
+
+            if val.endswith(".nii" or ".nii.gz"):
+                check_reorient = pe.Node(
+                    interface=afni.Resample(),
+                    name=f"reorient_{key}",
+                )
+
+                check_reorient.inputs.orientation = desired_orientation
+                check_reorient.inputs.outputtype = "NIFTI_GZ"
+
+                wf.connect(node, output, check_reorient, "in_file")
+                node = check_reorient
+                output = "out_file"
+                node_name = f"{key}_reorient"
```

Comment:
If we do this, this can just be
```suggestion
                check_reorient = rpool.reorient_node(key, set_data=False)
                wf.connect(node, output, check_reorient, "in_file")
                node = check_reorient
                output = "out_file"
```

Reply:
with the changes, I think this is not needed.